### DISCUSSION TITLE
How can I integrate PII Masking of LiteLLM?

### DISCUSSION THREAD

**Discussion by richard-313** [open-webui/open-webui#1015] [General]
Hi guys,
how can I integrate the PII Masking of LiteLLM into my Open-WebUI? 
https://litellm.vercel.app/docs/proxy/pii_masking

Any ideas?

**Comment by ishaan-jaff** [comment]
Hi @s72817 I'm the litellm maintainer. Can we hop on a call to better understand this? I can help with adding PII Masking 

Sharing a link to my cal for your convenience: https://calendly.com/d/4mp-gd3-k5k/litellm-1-1-onboarding-chat?month=2024-03
